src/utils/system.hpp

```diff
@@ -88,10 +88,12 @@ class System {
   // Navigation IMU axis
   int8_t axis;
   // Navigation kind of run
-  bool tube_run;
+  bool official_run;
   bool elevator_run;
   bool stationary_run;
   bool outside_run;
+  // Write acc,vel,pos to file
+  bool nav_write;
```

Comment:
Yeah I suppose, I just usually like having the names as short as possible. I don't like the enable tag because it makes it look like function imo. Would be best to just call it `nav_write_to_file` in that case, if it looks unintuitive atm.